plugins/queue/providers/rabbit_mq/RabbitMQPlugin.php

```diff
@@ -1,91 +1,107 @@
 <?php
-/**
- * @package plugins.rabbitMQ
- */
-class RabbitMQPlugin extends KalturaPlugin implements IKalturaPending, IKalturaObjectLoader, IKalturaQueuePlugin
-{
-	const PLUGIN_NAME = 'rabbitMQ';
-	const QUEUE_PLUGIN_NAME = 'queue';
-	const QUEUE_PLUGIN_VERSION_MAJOR = 1;
-	const QUEUE_PLUGIN_VERSION_MINOR = 0;
-	const QUEUE_PLUGIN_VERSION_BUILD = 0;
 	
-	/*
-	 * (non-PHPdoc)
-	 * @see IKalturaPlugin::getPluginName()
-	 */
-	public static function getPluginName()
-	{
-		return self::PLUGIN_NAME;
-	}
-	
-	/*
-	 * (non-PHPdoc)
-	 * @see IKalturaObjectLoader::getObjectClass()
+	/**
+	 * @package plugins.rabbitMQ
 	 */
-	public static function getObjectClass($baseClass, $enumValue)
+	class RabbitMQPlugin extends KalturaPlugin implements IKalturaPending, IKalturaObjectLoader, IKalturaQueuePlugin, IKalturaEnumerator
 	{
-		if($baseClass == 'QueueProvider' && (is_null($enumValue) || $enumValue == self::getRabbitMQProviderTypeCoreValue(RabbitMQProviderType::RABBITMQ)))
+		const PLUGIN_NAME = 'rabbitMQ';
+		const QUEUE_PLUGIN_NAME = 'queue';
+		const QUEUE_PLUGIN_VERSION_MAJOR = 1;
+		const QUEUE_PLUGIN_VERSION_MINOR = 0;
+		const QUEUE_PLUGIN_VERSION_BUILD = 0;
+		
+		/*
+		 * (non-PHPdoc)
+		 * @see IKalturaPlugin::getPluginName()
+		 */
+		public static function getPluginName()
 		{
-			if(!kConf::hasMap('rabbit_mq'))
-			{
-				throw new kCoreException("RabbitMQ configuration file (rabbit_mq.ini) wasn't found!");
-			}
-			$rabbitConfig = kConf::getMap('rabbit_mq');
-			
-			if(isset($rabbitConfig['multiple_dcs']) && $rabbitConfig['multiple_dcs'])
-			{
-				return 'MultiCentersRabbitMQProvider';
+			return self::PLUGIN_NAME;
+		}
+		
+		/*
+		 * (non-PHPdoc)
+		 * @see IKalturaObjectLoader::getObjectClass()
+		 */
+		public static function getObjectClass($baseClass, $enumValue)
+		{
+			if ($baseClass == 'QueueProvider' && (is_null($enumValue) || $enumValue == self::getRabbitMQProviderTypeCoreValue(RabbitMQProviderType::RABBITMQ))) {
+				if (!kConf::hasMap('rabbit_mq')) {

```

Comment:
i know it was like that before your code change but i would define this map name as well and fix the indentation

Reply:
Done